modules/system/lang/de/client.php

```diff
@@ -61,7 +61,11 @@
 
     'filter' => [
         'group' => [
-            'all' => 'Alle'
+            'all' => 'all',
```

Comment:
@datune Is this meant to be "alle"? I'm probably wrong, but just wanted to check :)

Would you also mind changing all "Alle" references in other `all` definitions, as these are all supposed to be lower-case?

Reply:
You‘re indeed wrong 😉 It is supposed to be like that.

Reply:
I sometimes am 😂 

Do you mind doing this for us?:
> Would you also mind changing all "Alle" references in other all definitions, as these are all supposed to be lower-case?

Reply:
Do I mind doing what? It has to be like this, I‘ve already applied this change in the last PR for this file.